src/displayapp/fonts/fonts.json

```diff
@@ -0,0 +1,90 @@
+{
+   "fonts": {
+      "jetbrains_mono_bold_20": {
+         "sources": [
+            {
+               "font": "JetBrainsMono-Bold.ttf",
+               "range": "0x20-0x7e, 0x410-0x44f"
+            },
+            {
+               "font": "FontAwesome5-Solid+Brands+Regular.woff",
+               "range": "0xf293, 0xf294, 0xf244, 0xf240, 0xf242, 0xf243, 0xf241, 0xf54b, 0xf21e, 0xf1e6, 0xf54b, 0xf017, 0xf129, 0xf03a, 0xf185, 0xf560, 0xf001, 0xf3fd, 0xf069, 0xf1fc, 0xf45d, 0xf59f, 0xf5a0, 0xf029, 0xf027, 0xf028, 0xf6a9, 0xf04b, 0xf04c, 0xf048, 0xf051, 0xf095, 0xf3dd, 0xf04d, 0xf2f2, 0xf024, 0xf252, 0xf569, 0xf201, 0xf06e, 0xf015"
+            }
+         ],
+         "bpp": 1,
+         "size": 20,
+         "patches": ["./fix_jetbrains_mono_bold_20_zero.py"]
+      },
+      "jetbrains_mono_42": {
+         "sources": [
+            {
+               "font": "JetBrainsMono-Regular.ttf",
+               "range": "0x25, 0x30-0x3a"
+            }
+         ],
+         "bpp": 1,
+         "size": 42
+      },
+      "jetbrains_mono_76": {
+         "sources": [
+            {
+               "font": "JetBrainsMono-Regular.ttf",
+               "range": "0x25, 0x2D, 0x2F, 0x30-0x3a"
+            }
+         ],
+         "bpp": 1,
+         "size": 76
+      },
+      "jetbrains_mono_extrabold_compressed": {
+         "sources": [
+            {
+               "font": "JetBrainsMono-ExtraBold.ttf",
+               "range": "0x30-0x3a"
+            }
+         ],
+         "bpp": 1,
+         "size": 80
+      },
+      "open_sans_light": {
+         "sources": [
+            {
+               "font": "open_sans_light.ttf",
+               "symbols": "0123456789"
+            }
+         ],
+         "bpp": 1,
+         "size": 150
+      },
+      "lv_font_sys_48": {
+         "sources": [
+            {
+               "font": "icons_sys_48.ttf",
+               "range": "0xe902, 0xe904-0xe907, 0xe90b-0xe90c"
+            }
+         ],
+         "bpp": 1,
+         "size": 48
+      },
+      "lv_font_navi_80": {
+         "sources": [
+            {
+               "font": "navigation.ttf",
+               "range": "0xe900-0xe929"
+            }
+         ],
+         "bpp": 2,
+         "size": 80,
+         "compress": true
+      }
+   },
+   "features": {
+      "hebrew": {
+         "jetbrains_mono_bold_20": [
+            {
+               "font": "SimpleCLM-Medium.ttf",
+               "range": "0x05D0-0x05EA,0x05F3,0x05F4"
+            }
+         ]
+      }
+   }
```

Comment:
Features should probably be inside the corresponding font section

Reply:
that will simplify the script, but will make the config less readable. like this the "features" can be clearly ignored by a human reading the config.

Reply:
Added commit which makes this change.

Reply:
it won't happen often, since most requests for font changes are either for a new language (which should be added in the same place as the hebrew example),
or for new watchfaces - which also use at most 1 font, and often a completely new one, and that need it (so it's not a toggle-able feature)